packages/client/examples/inlineClient.ts

```diff
@@ -0,0 +1,19 @@
+import { createCommonFromGethGenesis } from '@ethereumjs/common'
+import { eip4844GethGenesis } from '@ethereumjs/testdata'
+import { Config } from '../src/config.ts'
+import { createInlineClient } from '../src/util/inclineClient.ts'
+
+const main = async () => {
+  const client = await createInlineClient(
+    new Config({}),
+    createCommonFromGethGenesis(eip4844GethGenesis, {}),
+    {},
+    undefined,
+    true,
+  )
+  await client.start()
+  await client.stop()
+  process.exit(0)
```

Comment:
Doesn't this mess with the tests? If we run the examples test, doesn't this now kill the entire test process?

Reply:
So the examples workflow is going over the `inlineClient` run - see e.g. [here](https://github.com/ethereumjs/ethereumjs-monorepo/actions/runs/14441921077/job/40493895457?pr=3972), so this seems not to be an issue here? Do you see other side effects? Can vaguely remember that we had some exchange around this whole "how to exit tests" topic, but cannot fully remember.

Side note: exiting with `0` means the "success" case, sometimes one is mixing this up as well.

<img width="863" alt="grafik" src="https://github.com/user-attachments/assets/f4005aa9-b233-4e80-916a-432c21b9c177" />